lib/commands/app/create.ts

```diff
@@ -75,30 +86,57 @@ export default class AppCreateCmd extends Command {
 			AppCreateCmd,
 		);
 
-		const balena = getBalenaSdk();
-
-		// Create application
+		// Ascertain device type
 		const deviceType =
 			options.type ||
 			(await (await import('../../utils/patterns')).selectDeviceType());
-		let application: BalenaSDK.Application;
+
+		// Ascertain organization
+		const organization = options.organization || (await this.getOrganization());
+
+		// Create application
+		let application: Application;
 		try {
-			application = await balena.models.application.create({
+			application = await getBalenaSdk().models.application.create({
 				name: params.name,
 				deviceType,
-				organization: (await balena.auth.whoami())!,
+				organization,
```

Comment:
Following what Thodoris mentiones in Flowdock:
> https://www.flowdock.com/app/rulemotion/p-organizations/threads/c1PZ2OS_rNmIRI3UJqKbzv306k-
> I think the issue there is that the SDK is only checking the handle

And checking the SDK documentation for `application.create`:
> https://www.balena.io/docs/reference/sdk/node-sdk/#balena.models.application.create
> options.organization - handle (string) or id (number) of the organization that the application will belong to or null

Now I understand that this property is necessarily a handle, not a fancy name. In this case, given:

```
const organization = options.organization || (await this.getOrganization());
```

It follows that `options.organization` should be a handle, not a name. So we to document this, and provide a way of users querying the handles `balena orgs`, and/or add code to fetch orgs by name rather than handle, then extract the handle...

Reply:
I'd favour sticking to handles for everything (rather than names).  It seems like the orgs team are ok with exposing handles to the user.

Reply:
> It follows that options.organization should be a handle, not a name. So we would have to document this, and provide a way of users querying the handles, e.g. balena orgs,

:+1: